src/processout/translator.ts

```diff
@@ -155,6 +155,55 @@ module ProcessOut {
             "request.gateway.not-supported": "La pasarela de pago no es compatible.",
             "resource.invalid-type": "El recurso proporcionado no es válido. Debe ser un invoice, una suscripción o una solicitud de autorización.",
             "applepay.not-supported": "Su navegador o dispositivo actual no es compatible con Apple Pay.",
+        },
+        "pt": {
+            "default": "Ocorreu um erro: o pagamento foi recusado.",
+            "card.declined": "O cartão foi recusado.",
+            "card.expired": "O cartão fornecido expirou.",
+            "card.duplicate": "O pagamento não foi concluído. Por favor, tente novamente.",
+            "card.network-failed": "O pagamento não foi concluído. Por favor, tente novamente.",
+            "card.invalid": "O cartão fornecido é inválido.",
+            "card.invalid-name": "O nome do titular do cartão é inválido.",
+            "card.invalid-number": "O número do cartão é inválido.",
+            "card.invalid-date, card.invalid-expiry-date": "A data de validade do cartão é inválida.",
+            "card.invalid-month, card.invalid-expiry-month": "O mês de validade do cartão é inválido.",
```

Comment:
`"card.invalid-month, card.invalid-expiry-month"` 

Is this valid (I don't know), will it treat those as individual or one whole string?

Reply:
No, just checked the final result
It is one key
I will separate